src/test/java/edu/kis/powp/jobs2d/events/SelectTestFigureOptionListener.java

```diff
@@ -28,5 +29,13 @@ else if(e.getActionCommand().equals("Figure Jane")) {
 			AbstractDriverAdapter driver = new AbstractDriverAdapter(driverManager.getCurrentDriver());
 			FiguresJane.figureScript(driver);
 		}
+		else if(e.getActionCommand().equals("Rectangle")) {
+			AbstractDriverAdapter driver = new AbstractDriverAdapter(driverManager.getCurrentDriver());
+			ComplexCommandFactory.rectangle(driver).execute();
+		}
+		else if(e.getActionCommand().equals("Triangle")) {
```

Comment:
Menu action should not depend on its caption. Enum or separate listeners would be better. At least you should consider constants.